cmd/minishift/cmd/start_preflight.go

```diff
@@ -433,22 +439,22 @@ func checkHypervDriverSwitch() bool {
 
 // checkHypervDriverInstalled returns true if Hyper-V driver is installed
 func checkHypervDriverInstalled() bool {
-	// Check if Hyper-V's Virtual Machine Management Service is installed
-	_, err := exec.LookPath("vmms.exe")
-	if err != nil {
-		return false
-	}
-
-	// check to see if a hypervisor is present. if hyper-v is installed and enabled,
 	posh := powershell.New()
 
+	// check to see if a hypervisor is present. if hyper-v is installed and enabled,
 	checkHypervisorPresent := `@(Get-Wmiobject Win32_ComputerSystem).HypervisorPresent`
```

Comment:
Did not change this..  as `Get-WindowsOptionalFeature -FeatureName Microsot-Hyper-V-All -Online` requires an elevated shell